frontend/components/Employees/EmployeesListActions.jsx

```diff
@@ -0,0 +1,37 @@
+import React, {useState} from 'react';
+import { SendIcon, EditIcon, LockIcon } from 'components/Icons';
+
+const EmployeesListActions = ({handleEdit}) => {
+  const [isHover, setIsHover] = useState({});
+  const icons = {
+    send: SendIcon,
+    edit: EditIcon,
+    lock: LockIcon
+  };
+
+  const setHover = (component, value) => {
+    setIsHover(prevState => ({...prevState, [component]: value}));
+  }
+
+  const getIconComponent = (name) => {
+    const Icon = icons[name];
+    return <Icon stroke={isHover[name] ? "red" : "none"}
```

Comment:
I suggest doing it with styles, 
for example, you have such icon with className:
<Icon className="icon"/>

in your style file, you can do this
.icon {
 // some styles 
}

.icon:hover {
  .. another styles
}